opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -1800,6 +1800,47 @@ namespace detail {
             }
         }
 
+        /// \brief Compute the L-infinity norm of a vector
```

Comment:
When I read the code it comes to me naturally (stream of programming conciousness?), it takes me 10 seconds to comment, and it takes 10 seconds to fix. Code ends up better and more readable. I do think it is quite appropriate!